src/components/availability-panel/drag-drop-context/DraggableComponent.tsx

```diff
@@ -33,6 +33,14 @@ const DraggableComponent: FC<draggableProps> = ({
   title = '',
 }) => {
   const draggableIds = useContext(disableDrag);
+  const [draggableImageUrl, setDraggableImageUrl] = useState(DUMMY_PROFILE);
+  const draggableImageOnError = useCallback(() => setDraggableImageUrl(DUMMY_PROFILE), []);
```

Comment:
You don't need to use `useCallback` here, as the cost to save recreation of the onErrorHandler function is not big enough.